### PR TITLE
[16.0][MIG] stock_quant_manual_assign

### PR DESCRIPTION
**Pull request by alexis-via**
Fix bug #1527  when using a unit of measure on stock.move.line which is different from product's unit of measure
Fix bug #1460 set "Others reserved" as readonly

### CODE REVIEW

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 1]
```diff
@@ -137,45 +137,38 @@ class AssignManualQuantsLines(models.TransientModel):
...
     )
     # This is not correctly shown as related or computed, so we make it regular
     on_hand = fields.Float(
         readonly=True,
         digits="Product Unit of Measure",
     )
-    reserved = fields.Float(string="Others Reserved", digits="Product Unit of Measure")
+    reserved = fields.Float(
+        string="Others Reserved", digits="Product Unit of Measure", readonly=True
+    )
```
Comment (sebalix) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py:171]: This deserves to be in its own commit (outside of the migration one) to be easily backported in previous versions.

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def assign_quants(self):
         if move.picking_type_id.auto_fill_qty_done:
             # Auto-fill all lines as done
             for ml in move.move_line_ids:
-                ml.qty_done = ml.product_qty
+                ml.qty_done = ml.reserved_uom_qty
```
Comment (sebalix) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py:67]: Same, to put in its own commit so we can backport it (even if the field has changed in 16.0).

Comment (rousseldenis) [reply stock_quant_manual_assign/wizard/assign_manual_quants.py:67]: @sebalix This is not meant to be backported as just introduced in v16

File: stock_quant_manual_assign/tests/test_stock_quant_manual_assign.py [hunk 3]
```diff
@@ -2,84 +2,88 @@
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 from odoo.exceptions import ValidationError
+from odoo.tests import tagged
 from odoo.tests.common import TransactionCase
 
 
+@tagged("post_install", "-at_install")
```
Comment (rousseldenis) [inline stock_quant_manual_assign/tests/test_stock_quant_manual_assign.py:9]: @alexis-via Is this really needed ?

File: stock_quant_manual_assign/wizard/assign_manual_quants.py (hunk 1 above)
Comment (rousseldenis) [reply stock_quant_manual_assign/wizard/assign_manual_quants.py:171]: @alexis-via Is the readonly meant to be backported ?

**Comment by florenciafrigieri2** [comment]
@alexis-via Hello! Can you please review this PR? Thanks!

**Comment by pedrobaeza** [comment]
@alexis-via can you please answer the questions and comments?

**Comment by pedrobaeza** [comment]
Superseded by #1707
